test/suites/basic.sh

```diff
@@ -197,23 +197,28 @@ test_basic_usage() {
   #   1. Basic built-in alias functionality
   [ "$(lxc ls)" = "$(lxc list)" ]
   #   2. Basic user-defined alias functionality
+  # shellcheck disable=SC1117
   printf "aliases:\n  l: list\n" >> "${LXD_CONF}/config.yml"
   [ "$(lxc l)" = "$(lxc list)" ]
   #   3. Built-in aliases and user-defined aliases can coexist
   [ "$(lxc ls)" = "$(lxc l)" ]
   #   4. Multi-argument alias keys and values
+  # shellcheck disable=SC1117
   printf "  i ls: image list\n" >> "${LXD_CONF}/config.yml"
   [ "$(lxc i ls)" = "$(lxc image list)" ]
   #   5. Aliases where len(keys) != len(values) (expansion/contraction of number of arguments)
+  # shellcheck disable=SC1117
   printf "  ils: image list\n  container ls: list\n" >> "${LXD_CONF}/config.yml"
   [ "$(lxc ils)" = "$(lxc image list)" ]
   [ "$(lxc container ls)" = "$(lxc list)" ]
   #   6. User-defined aliases override built-in aliases
+  # shellcheck disable=SC1117
   printf "  cp: list\n" >> "${LXD_CONF}/config.yml"
   [ "$(lxc ls)" = "$(lxc cp)" ]
   #   7. User-defined aliases override commands and don't recurse
   lxc init testimage foo
   LXC_CONFIG_SHOW=$(lxc config show foo --expanded)
+  # shellcheck disable=SC1117
```

Comment:
All of those printfs should just be replaced with "echo" and the trailing \n be removed from the string.

Reply:
I'll replace some of them with `echo`. The ones containing a newline in the middle won't change, otherwise shellcheck will complain with

- `SC2039: In POSIX sh, echo flags are undefined.` (when using `echo -e "foo\\nbar"`)
- `SC2028: echo won't expand escape sequences. Consider printf.` (when using `echo "foo\\nbar"`)